File: Data Structures & Algorithms/sort-colors/submission-0.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) 
    {
        int count[3] = {};
        
        for (int x : nums) 
            count[x]++;
        
        int i = 0;
        
        for (int color = 0; color < 3; color++)
        {
            while (count[color]--) 
            {
                nums[i++] = color;
            }
        }
    }
};

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        count ={}
        for n in nums:
            if n not in count :
                count[n] = 0
            count[n]+=1
        
        top = max(count, key =lambda k:count[k])
        return top
        